src/components/EasyPill/EasyPill.test.tsx

```diff
@@ -2,17 +2,29 @@ import React from 'react';
 import { mount, shallow } from 'enzyme';
 
 import Pill from '../Pill';
-import EasyPill from './';
+import EasyPill from '.';
 
 describe('EasyPill', () => {
+  const defaultActions = [
+    {
+      label: 'myAction',
+      onClick: () => {},
+    },
+  ];
   it('renders', () => {
-    shallow(<EasyPill onDelete={() => {}}>EasyPill</EasyPill>);
+    shallow(
+      <EasyPill actions={defaultActions} onDelete={() => {}}>
```

Comment:
I saw Dan's comment - he's right, `actions` wasn't required before. We should leave it that way, and then this won't be necessary.